packages/back-end/src/scim/users/listUsers.ts

```diff
@@ -0,0 +1,73 @@
+import { Response } from "express";
+import { parse, filter } from "scim2-parse-filter";
+import { expandOrgMembers } from "../../services/organizations";
+import { ScimListRequest, ScimUser } from "../../../types/scim";
+
+export const START_INDEX_DEFAULT = 1;
+export const COUNT_DEFAULT = 20;
+
+export async function listUsers(req: ScimListRequest, res: Response) {
+  const { startIndex, count, filter: filterQuery } = req.query;
+
+  const queryOptions = {
+    startIndex: startIndex ? parseInt(startIndex) : START_INDEX_DEFAULT,
+    count: count ? parseInt(count) : COUNT_DEFAULT,
+  };
+
+  const org = req.organization;
+
+  const expandedMembers = await expandOrgMembers(org.members);
+
+  // TODO: return inactive users too for self-hosted to adhere to SCIM (users not in the org)
```

Comment:
This comment makes me a little worried.  An organization's API key only gives it access to the organization.  Returning members that are not in the org is not allowed